app/views/layouts/app_no_container.html.erb

```diff
@@ -8,6 +8,7 @@
 
 
     <!-- Import Stylesheets -->
+    <link href="https://use.fontawesome.com/releases/v5.0.8/css/all.css" rel="stylesheet">
```

Comment:
You're using two different CDNs, which kinda defeats the purpose, settle on one.